DicomUltrasoundPlugin/DicomUltrasoundPlugin.py

```diff
@@ -565,10 +565,15 @@ def load(self,loadable):
     else:
       loadedNode = self.loadPhilips4DUSAsSequence(loadable)
 
+    # Save DICOM SOP instance UID into the sequence so DICOM metadata can be retrieved later if needed
+    if loadedNode:
+      loadedNode.SetAttribute('DICOM.instanceUIDs', slicer.dicomDatabase.instanceForFile(loadable.files[0]))
+
     # Show sequence browser toolbar if a sequence has been loaded
     if loadedNode and loadedNode.IsA('vtkMRMLSequenceNode'):
       sequenceBrowserNode = slicer.modules.sequences.logic().GetFirstBrowserNodeForSequenceNode(loadedNode)
       if sequenceBrowserNode:
+        loadedNode.SetAttribute('DICOM.instanceUIDs', loadedNode.GetAttribute('DICOM.instanceUIDs'))

```

Comment:
```suggestion
        sequenceBrowserNode.SetAttribute('DICOM.instanceUIDs', loadedNode.GetAttribute('DICOM.instanceUIDs'))
```

Reply:
The sequence browser node can browse many different images, so in general the DICOM instance UIDs cannot be saved into that.

Instead, the attribute need to be set on the volume node in the sequence (and those attributes are copied to the proxy node).

Reply:
```suggestion
        volumeNode = browserNode.GetProxyNode(loadedNode)
        volumeNode.SetAttribute('DICOM.instanceUIDs', loadedNode.GetAttribute('DICOM.instanceUIDs'))
```

Reply:
Thanks for the suggestion.